app/src/main/java/org/traccar/manager/commandhandler/PositionPeriodicCommandHandler.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2016 Gabor Somogyi (gabor.g.somogyi@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.traccar.manager.commandhandler;
+
+import android.content.res.Resources;
+import android.view.View;
+import android.widget.LinearLayout;
+import android.widget.NumberPicker;
+import android.widget.Spinner;
+
+import org.traccar.manager.R;
+import org.traccar.manager.model.Command;
+
+public class PositionPeriodicCommandHandler implements CommandHandler {
+    private Resources resources;
+    private LinearLayout frequencyGroup;
+    private NumberPicker frequencyNumberPicker;
+    private LinearLayout unitGroup;
+    private Spinner unitSpinner;
+
+    public PositionPeriodicCommandHandler(View view, Resources resources) {
+        frequencyGroup = (LinearLayout) view.findViewById(R.id.frequencyGroup);
+        frequencyNumberPicker = (NumberPicker) view.findViewById(R.id.frequency);
+        unitGroup = (LinearLayout) view.findViewById(R.id.unitGroup);
+        unitSpinner = (Spinner) view.findViewById(R.id.unit);
+
+        frequencyNumberPicker.setMinValue(0);
```

Comment:
The default limit is [0, 0] so something must be set. The question is where and what. When I've tries Integer.MAX_VALUE it was ugly. Suggestions?